src/main/java/pl/sda/javapoz/security/MyUserPrincipal.java

```diff
@@ -0,0 +1,51 @@
+package pl.sda.javapoz.security;
+
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.userdetails.UserDetails;
+import pl.sda.javapoz.model.User;
+
+import java.util.Collection;
+
+public class MyUserPrincipal implements UserDetails {
+
+    private User user;
+
+    public MyUserPrincipal(User user) {
+        this.user = user;
+    }
+
+    @Override
+    public Collection<? extends GrantedAuthority> getAuthorities() {
+        return null;
```

Comment:
To jest na bank do zmiany, bo nie mamy teraz ról.

Reply:
nie mamy ról, ale i tak korzystając z UserDetails i tak to musi być